src/services/user/services/AdminUsers.js

```diff
@@ -219,9 +218,9 @@ const adminHookGenerator = (kind) => ({
 		all: [authenticate('jwt')],
 		find: [hasSchoolPermission(`${kind}_LIST`)],
 		get: [hasSchoolPermission(`${kind}_LIST`)],
-		create: [hasSchoolPermission(`${kind}_CREATE`), blockDisposableEmail('email')],
-		update: [hasSchoolPermission(`${kind}_EDIT`)],
-		patch: [hasSchoolPermission(`${kind}_EDIT`)],
+		create: [hasSchoolPermission(`${kind}_CREATE`), checkUniqueEmail, blockDisposableEmail('email')],
+		update: [hasSchoolPermission(`${kind}_EDIT`), checkUniqueEmail],
+		patch: [hasSchoolPermission(`${kind}_EDIT`), checkUniqueEmail],
```

Comment:
IIRC, checking the email is part of this service already. cc: @CordlessWool

Reply:
is included in the service

Reply:
Yes... Someone added the checkMail() in the meantime 

- https://github.com/hpi-schul-cloud/schulcloud-server/pull/1756/files

In this case, we can close this PR without merging.

Reply:
I have removed the Email check from AdminUsers